pysoa/server/settings.py

```diff
@@ -59,7 +59,13 @@ class ServerSettings(SOASettings):
                 ),
                 'filters': fields.SchemalessDictionary(
                     key_type=fields.UnicodeString(),
-                    value_type=fields.Dictionary({'name': fields.UnicodeString()}, optional_keys=('name', )),
+                    value_type=fields.Dictionary(
+                        {
+                            '()': fields.Anything(description='The optional filter class'),
```

Comment:
why `'()'` as a field name?

Reply:
Because that's the way Python Logging config works. To create a logging filter of a specific class, you use `'()'` as the field name. I have no idea why. It's ugly as sin. But it's part of Python; not in our control.